Triangulation_2/doc/Triangulation_2/CGAL/Triangulation_2_projection_traits_3.h

```diff
@@ -0,0 +1,79 @@
+namespace CGAL{
+
+/*!
+ \ingroup PkgTriangulation2TriangulationClasses
+
+  The class `Triangulation_2_projection_traits_3` is an extension of the `Projection_traits_xy_3`,
+  `Projection_traits_xz_3` and `Projection_traits_yz_3` traits classes,that allows the use
+  of 2D algorithms on the projections of 3D data on any plane.
+
+  \cgalHeading{Parameters}
+  The template parameter `K` has to
+  be instantiated by a model of the `K` concept.
+  `Triangulation_2_projection_traits_3` uses types
+  and predicates defined in `K`.
+
+\cgalModels `TriangulationTraits_2`
+\cgalModels `DelaunayTriangulationTraits_2`
+\cgalModels `ConstrainedTriangulationTraits_2`
+
+ */
+template <class K>
+class Triangulation_2_projection_traits_3
+{
+public:
+  ///\name Types
+  ///@{
+
+  //!
+  typedef typename K::Point_3     Point_2;
+  //!
+  typedef typename K::Segment_3   Segment_2;
+  //!
+  typedef typename K::Triangle_3  Triangle_2;
+  //!
+  typedef typename K::Line_3      Line_2;
```

Comment:
Don't know if there is much worth to documenting those: they are not used anywhere in the doc of this class (except `Point_2`).